test/conformance/serial/prometheus.go

```
@@ -28,6 +28,14 @@ const openshiftPtpMetricPrefix = "openshift_ptp_"
 //	     "metric" : {
 //	       "pod" : "mm-pod1",
 //	}}]}}
+type queryOutput struct {
```

golangci-lint (2.12.2)

[error] type queryOutput is unused

(unused)